options/internal/m68k-include/mlibc/thread.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <stdint.h>
+#include <mlibc/internal-sysdeps.hpp>
+#include <mlibc/tcb.hpp>
+#include <bits/ensure.h>
+
+namespace mlibc {
+
+extern "C" void *__m68k_read_tp();
+
+inline Tcb *get_current_tcb() {
+	// On m68k, the TCB is below 0x7000 below the thread pointer.
```

Comment:
nit: below repeated twice

Reply:
I worded this in a very ugly way, it was supposed to convey that
```
+-----+---------------|------+
| TCB |  0x7000 bytes | more |
+-----+---------------+------+
                      ^
                      TP
```
but could benefit from being reworded.